server/src/main/java/org/apache/druid/discovery/DiscoveryDruidNode.java

```diff
@@ -35,22 +40,29 @@
  */
 public class DiscoveryDruidNode
 {
+  private static final Logger LOG = new Logger(DiscoveryDruidNode.class);
+  private static final TypeReference<Map<String, Object>> RAW_DRUID_SERVICE_TYPE =
```

Comment:
I don't see any reference to this static variable, do I miss something?

Reply:
Good catch! It is a left over that I forgot to delete after I cleaned up my PR. I will delete it.